lib/bug_report_templates/rails_5_master.rb

```diff
@@ -51,32 +49,28 @@ class ApplicationController < ActionController::Base
 
 class TestApp < Rails::Application
   config.root = File.dirname(__FILE__)
-  config.logger = Logger.new(STDOUT)
-
+  config.session_store :cookie_store, key: "cookie_store_key"
   secrets.secret_token = 'secret_token'
   secrets.secret_key_base = 'secret_key_base'
 
   config.eager_load = false
+  config.logger = Logger.new($stdout)
+  Rails.logger  = config.logger
 end
 
-# create models
 class YourModel < ActiveRecord::Base
 end
 
-# configure active_admin
 ActiveAdmin.setup do |config|
-  # Authentication disabled by default. Override if necessary
+  # Authentication disabled by default. Override if necessary.
```

Comment:
Because the others are self explanatory, ActiveAdmin.setup, Rails.app.initialize, etc. I liked having this here because its common to have authentication for AA so its a nice heads up that as a convenience its disabled.